src/Ticket.php

```diff
@@ -1259,12 +1259,12 @@ public function prepareInputForUpdate($input)
                     switch ($a) {
                         case 'user':
                              $additionalfield           = '_additional_' . $t . 's';
-                             $input[$additionalfield][] = ['users_id' => $input['_' . $a . 's_id_' . $t]];
+                             $input[$additionalfield] = ['users_id' => $input['_' . $a . 's_id_' . $t]];
```

Comment:
Unless I am wrong, if you have multiple users of same type, only the last one will be sent to BR, as other will be erased.

Reply:
Might been the case before PR #11728 
But now I seen that array appear on _groups_id_assign (for example and in turn here).
But you are right there are still cases where just string is present in _groups_id_assign so this needs to be handled too...

Reply:
I guess #11812 should fix it.

Reply:
Yeah this one fixes the issue with scalars. But still, I can't get Actors behavior as I intend to...When Escalade plugin is enabled to keep just one group in "Assigned To" (eg. "Remove old assign group on new group assign"). Plugin does this but later on  this is revered by CommonITILObject->post_updateItem() which calls CommonITILObject->updateActors() and again adds old group based on Tickets->input['_actors'] array...

Thinking how to fix this..